Wikimedia.md

```diff
@@ -1,64 +1,174 @@
-# Wikimedia
+Wikimedia
 
-## Allgemeines
-https://ideen.wikimedia.de/projekt/projekt-starten/progress/194
+[https://ideen.wikimedia.de/projekt/projekt-starten/progress/194](https://ideen.wikimedia.de/projekt/projekt-starten/progress/194)
 
-## Projektvorstellung
+Mail der Wikimedia Deutschland:
```

Comment:
Trenne mal bitte Aktionen von Ergebnis. Die Email kann in ein Ticket MIT Handlungsanweisung.